.storybook/preview.js

```diff
@@ -8,4 +8,5 @@ export const parameters = {
       date: /Date$/,
     },
   },
+  layout: 'fullscreen',
```

Comment:
It is probably more useful that this is done at the story level than across the board. As though a nav might be better with the `fullscreen` layout we get a better picture of most other components with space around them.